src/PlayerList.tsx

```diff
@@ -21,9 +21,13 @@ function PlayerList({
 }: PlayerListProps): JSX.Element {
   const { metrics } = timeline[currentStep];
   // TODO: Toggle sorting by kills / score / default?
-  const sortedPlayers = [...players].sort(
-    (a, b) => metrics[b.index].score - metrics[a.index].score
+  const alivePlayers = players.filter(
+    (a) => metrics[a.index].score !== DEAD_SCORE
   );
+  const deadPlayers = players.filter((a) => !alivePlayers.includes(a));
+  alivePlayers.sort((a, b) => metrics[b.index].score - metrics[a.index].score);
+  deadPlayers.sort((a, b) => (b.killedStep ?? 0) - (a.killedStep ?? 0));
```

Comment:
I wasn't sure if using `.killedStep!` was actually fine here. It should be, the player is dead so they *should* have a `killedStep`

Reply:
I think using `?? 0` or `|| 0` is fine, as it covers all cases safely

I do think this should be doable with a single sort call, possibly worth extracting the sort function to be standalone and multi line so it's easy to see what's going on

Reply:
> I do think this should be doable with a single sort call

It's doable, but do you mean doable using no if statements? I thought filter calls would be better as there would be less elements to sort.

Reply:
I mean without splitting into dead/alive first

I'd use if statements in the sort function